bin.src/certifyCalibration.py

```diff
@@ -67,11 +67,11 @@
     butler = Butler(args.root, run=args.inputCollection)
```

Comment:
In the butler root above can you say "URI to butler" or something because it's not always a path.